src/sentry/demo/settings.py

```diff
@@ -0,0 +1,18 @@
+from datetime import timedelta
+from django.conf import settings
+
+from sentry.conf.server import *  # NOQA
+
+"""
+To get this file to load, add the follwing to your sentry.conf.py file:
+
+from sentry.demo.settings import *
+
+"""
+
+if settings.DEMO_MODE:
```

Comment:
It probably makes sense to set `DEMO_MODE = True` in this file. We're always going to be in demo mode if we have loaded this config file, I think?

You can also remove this check, for the same reason.